server/methods/alexaApiCall.js

```diff
@@ -0,0 +1,24 @@
+import { Meteor } from 'meteor/meteor';
+import { HTTP } from 'meteor/http';
+
+import { settings } from '../../app/settings';
+
+Meteor.methods({
+	registerAlexaUser(serverurl, servername, userid, token) {
+		this.unblock();
```

Comment:
You don't need this unblock.
And should check if there is an user logged before proceed